app/models/user_preference.rb

```diff
@@ -38,7 +38,8 @@ class UserPreference < ApplicationRecord
 
   WORKDAYS_FROM_MONDAY_TO_FRIDAY = [1, 2, 3, 4, 5].freeze
 
-  THEMES = %w[light light_high_contrast dark dark_high_contrast sync_with_os].freeze
+  COLOR_MODES = %w[light dark].freeze
+  THEMES = COLOR_MODES + %w[sync_with_os].freeze
```

Comment:
I'd prefer to use symbols here. Also, `THEMES` itself isn't frozen.

```suggestion
  COLOR_MODES = %i[light dark].freeze
  THEMES = [*COLOR_MODES, :sync_with_os].freeze
```